src/ptvsd/server/api.py

```diff
@@ -13,10 +13,19 @@
 import ptvsd
 from ptvsd.common import log, options as common_opts
 from ptvsd.server import options as server_opts
+from ptvsd.common.compat import queue
 from _pydevd_bundle.pydevd_constants import get_global_debugger
 from pydevd_file_utils import get_abs_path_real_path_and_base_from_file
 
 
+_ADAPTER_START_TIMEOUT = 10
+_dap_listener = None
```

Comment:
These two are unused now.

Reply:
Renamed one of then to use with `get()`. The other one was unnecessary.